packages/instant/src/components/amount_input.tsx

```diff
@@ -0,0 +1,42 @@
+import { BigNumber } from '@0xproject/utils';
+import * as _ from 'lodash';
```

Comment:
Per convo, added task here to look into if we're shipping all of lodash with instant: https://app.asana.com/0/710162581425378/860502394999228/f